### PR TITLE
Add shisui's story blog

### LINKED ISSUE #17: blog post idea: shisui's story
Introducing how Shisui was born

### PR DESCRIPTION
**Pull request by GrapeBaBa**
Fix #17

### CODE REVIEW

**Review by KolbyML — APPROVED** [review]
I think this looks good :rocket:

File: pages/posts/shisui-story.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# What is shisui
+Shisui is an Ethereum portal client written in Go language.
+The name is inspired by Uchiha Shisui from the anime Naruto, who is renowned as "Shisui of the Body Flicker".
+
+# How shisui was born
+When I was learning about data availability, I saw the portal network protocol from the Ethereum website.
+Through the documents and videos on the portal network website, I further learned about the decentralized vision of the portal network.
+I thought it was great. At the same time, during my nearly ten years of open-source work, I have always hoped to build an open-source system that can be used by many people.
+When I found that there is currently no Go language version of the portal client, I decided to start building shisui.
+At the same time, Dr. Zhou Qi from EthStorage wanted to use the portal client in their web3:// protocol. They were also interested in implementing the portal client in the Go language, so we established contact.
```
Comment (pipermerriam) [inline pages/posts/shisui-story.mdx:18]:
```suggestion
At the same time, Dr. Zhou Qi from EthStorage wanted to use the portal client in their web3:// protocol. They were also interested in implementing the portal client in the Go language, so we joined forces.
```

**Review by pipermerriam — APPROVED** [review]
Glad to have your team on the project.  Thanks for getting this written and introducing your client to the world.  :shipit:

**Comment by jmcook1186** [comment]
Do you have a website, github or any docs you could link out to @GrapeBaBa ?

**Review by jmcook1186 — APPROVED** [review]
thanks!

**Pull request merged**